src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,113 @@
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement getStatement =
+                         connection.prepareStatement(getRequest)) {
+            getStatement.setLong(1, id);
+            ResultSet resultSet = getStatement.executeQuery();
+            if (resultSet.next()) {
+                return Optional.of(createManufacturer(resultSet, id));
+            }
+            return Optional.of(new Manufacturer());
+        } catch (SQLException e) {
+            throw new RuntimeException("Can't get manufacturer by id: " + id + " from DB.", e);
+        }
+    }
+
+    @Override
+    public List<Manufacturer> getAll() {
+        List<Manufacturer> manufacturerList = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                 Statement getAllStatement = connection.createStatement()) {
+            ResultSet resultSet = getAllStatement
+                    .executeQuery("SELECT * FROM manufacturers where is_deleted = false");
+            while (resultSet.next()) {
+                Long id = resultSet.getObject("id", Long.class);
+                manufacturerList.add(createManufacturer(resultSet, id));
+            }
+        } catch (SQLException e) {
+            throw new RuntimeException("Can't get all manufacturer from DB", e);
+        }
+        return manufacturerList;
+    }
+
+    @Override
+    public Manufacturer update(Manufacturer manufacturer) {
+        String updateRequest = "UPDATE manufacturers SET name = ?, country = ?  WHERE id = ?";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement updateData =
+                         connection.prepareStatement(updateRequest,
+                             Statement.RETURN_GENERATED_KEYS)) {
+            updateData.setLong(3, manufacturer.getId());
+            updateData.setString(1, manufacturer.getName());
+            updateData.setString(2, manufacturer.getCountry());
+            updateData.executeUpdate();
+            return manufacturer;
+        } catch (SQLException e) {
+            throw new RuntimeException("Can't update manufacturer in DB", e);
+        }
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        String deleteRequest = "UPDATE manufacturers SET is_deleted = true where id = ?";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement deleteById =
+                        connection.prepareStatement(deleteRequest,
+                                Statement.RETURN_GENERATED_KEYS)) {
+            deleteById.setLong(1, id);
+            return deleteById.executeUpdate() >= 1;
+        } catch (SQLException e) {
+            throw new RuntimeException("Can't delete manufacturer by id: " + id + " from DB.", e);
+        }
+    }
+
+    private Manufacturer createManufacturer(ResultSet resultSet, Long id) throws SQLException {
```

Comment:
```suggestion
    private Manufacturer createManufacturer(ResultSet resultSet) throws SQLException {
```
id краще теж витягнути з ResultSet